tests/unittest/backend/test_gemm_rrr_bias.py

```diff
@@ -0,0 +1,157 @@
+#  Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Instead of adding this file, could you add new test functions to the existing [test_gemm_bias.py](https://github.com/facebookincubator/AITemplate/blob/main/tests/unittest/ops/test_gemm_bias.py)? You can use the existing test functions for `gemm_rcr_bias` as an example (e.g., copy, rename, and adapt). Also, no need to add `*_sm90` tests, as we're skipping CUTLASS 3.x support at this point.

Reply:
OK, work in progress.